main.cpp

```diff
@@ -66,6 +68,11 @@ int main(int argc, char *argv[]) {
     a.setApplicationName("KShare");
     a.setOrganizationName("ArsenArsen");
     a.setApplicationVersion("4.1");
+    
+    QLockFile lockFile(QDir::temp().absoluteFilePath("arsenarsen.kshare.lock"));
```

Comment:
Doesn't look different to me :p

Reply:
See my comment, I can't believe I forgot it